src/main/webapp/approval.html

```diff
@@ -0,0 +1,74 @@
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    
+    <title>Mentor Approval | Mintern</title>
+  </head>
+  <body onload="loadApproval()">
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <!-- Logo -->
+      <a class="navbar-brand" href="#">
+        <img src="images/mintern-logo.png" height="50" alt="Mintern logo">
+      </a>
+      <button
+          class="navbar-toggler"
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarToggler"
+          aria-controls="navbarToggler"
+          aria-expanded="false"
+          aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarToggler">
+        <ul class="navbar-nav mt-2 mt-lg-0 ml-auto">
+          <!-- Logout Button -->
+          <div id="login"></div>
+          <!-- Notifications Dropdown -->
+          <li class="nav-item dropdown">
+            <!-- Notifications Icon -->
+            <a class="nav-link dropdown-toggle" href="#" id="notificationsDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              <svg width="30px" height="30px" viewBox="0 0 16 16" class="bi bi-inbox-fill" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" d="M3.81 4.063A1.5 1.5 0 0 1 4.98 3.5h6.04a1.5 1.5 0 0 1 1.17.563l3.7 4.625a.5.5 0 0 1-.78.624l-3.7-4.624a.5.5 0 0 0-.39-.188H4.98a.5.5 0 0 0-.39.188L.89 9.312a.5.5 0 1 1-.78-.624l3.7-4.625z"/>
+                <path fill-rule="evenodd" d="M.125 8.67A.5.5 0 0 1 .5 8.5h5A.5.5 0 0 1 6 9c0 .828.625 2 2 2s2-1.172 2-2a.5.5 0 0 1 .5-.5h5a.5.5 0 0 1 .496.562l-.39 3.124a1.5 1.5 0 0 1-1.489 1.314H1.883a1.5 1.5 0 0 1-1.489-1.314l-.39-3.124a.5.5 0 0 1 .121-.393z"/>
+              </svg>
+            </a>
+            <!-- Notifications Dropdown -->
+            <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown" id="dropdown-override">
+              <!-- Container for addition of notifications. -->
+              <ul class="list-group" id="inbox-dropdown"></ul>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <!-- Approval -->
+    <div class="content m-5">
+      <div class="container text-center">
+        <h1 class="signup-title">Mentor Approval</h1>
+        <h2 class="signup-subtitle text-muted">Below you can find the information related to internship experience of: <span id="username"></span></h2> 
+        <h2 class="signup-subtitle text-muted">Please verify their validity.</h2> 
```

Comment:
I wonder if in the future you want to give more details / instructions for how to verify a mentor. Might be worth adding a TODO to do so.

Reply:
They only have to read the information and press and "Approve" or "Reject" button, so I changed that line to: `Please read through this information and verify its validity.`. Do you think they need more information?

Reply:
As a user (and I guess this doesn't matter for an admin which is why I'm suggesting a TODO), I wouldn't necessarily understand how I'm supposed to verify its validity, which is why I'm suggesting adding some instructions / suggestions for that!